.github/workflows/pytest.yml

```diff
@@ -97,11 +97,11 @@ jobs:
           pip install --no-deps --editable .
           make pytest-unit
 
-      - name: Upload coverage
+      - name: Upload unit tests coverage artifact
         uses: actions/upload-artifact@v4
         with:
           name: coverage-unit
-          path: coverage.xml
+          path: ./*coverage*
 
   ci-integration:
```

Comment:
The integration tests are still skipped even with a manual `workflow_dispatch`, so it seems like this conditional isn't working correctly.